core/ArchiveProcessor/Loader.php

```diff
@@ -152,15 +152,15 @@ private function prepareArchiveImpl($pluginName)
                     return $data;
                 }
 
-                list($idArchives, $visits, $visitsConverted) = $data;
+                list($idArchives, $visits, $visitsConverted, $existingArchives) = $data;
```

Comment:
fyi not sure if that was meant to be `$foundRecords`

Reply:
:+1: thanks for catching that